### PR TITLE
examples: add gimbal device tester

### PR DESCRIPTION
**Pull request by julianoes**
This is work in progress to test a gimbal device according to the gimbal v2 protocol, see: https://mavlink.io/en/services/gimbal_v2.html#how-to-implement-the-gimbal-device-interface

### CODE REVIEW

File: examples/gimbal_device_tester/gimbal_device_tester.cpp [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+using namespace mavsdk;
+
+static void usage(const std::string& bin_name);
+static bool request_gimbal_device_information(MavlinkPassthrough& mavlink_passthrough);
+static float degrees(float radians);
+static float radians(float radians);
+
+static void send_autopilot_state_for_gimbal_device(MavlinkPassthrough& mavlink_passthrough);
+static  void send_gimbal_device_set_attitude(MavlinkPassthrough& mavlink_passthrough);
```
Comment (MatejFranceskin) [inline examples/gimbal_device_tester/gimbal_device_tester.cpp:22]: 2 spaces between static  void...
(At least I found something ;-)

**Review by MatejFranceskin — APPROVED** [review]
Looks good to me.

**Comment by julianoes** [comment]
Let's merge this asap and then iterate on it.

File: examples/gimbal_device_tester/gimbal_device_tester.cpp [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+
+        if (fut.wait_for(std::chrono::seconds(2)) != std::future_status::ready) {
+            std::cout << "FAIL\n";
+            std::cout << "-> no device found" << std::endl;
+            return 1;
+        }
+        mavsdk.register_on_discover(nullptr);
+    }
+
+    System& system = mavsdk.system();
```
Comment (JonasVautherin) [inline examples/gimbal_device_tester/gimbal_device_tester.cpp:537]: This is deprecated, I think that's why the CI complains

**Review by JonasVautherin — CHANGES_REQUESTED** [review]

**Pull request merged**